data_steward/cdr_cleaner/manual_cleaning_rules/dc732_suppress.py

```diff
@@ -0,0 +1,598 @@
+    backup_rows_to_suppress(tables_to_suppress_df, rows_dest_table)
+
+    # delete rows
+    all_delete_queries = get_delete_queries(rows_dest_table)
+    delete_query_jobs = run_delete_queries(all_delete_queries)
+    print_jobs(delete_query_jobs)
+
+
+def get_arg_parser():
+    """
+    Get the parser for command line arguments
+    :return: the parser
+    """
+    parser = argparse.ArgumentParser()
+    parser.add_argument(
+        '-p',
+        '--project_id',
+        action='store',
+        dest='project_id',
+        help='Identifies the project containing the datasets to retract from',
+        required=True)
+    subparsers = parser.add_subparsers(dest='cmd')
+    setup_parser = subparsers.add_parser('setup')
+    setup_parser.add_argument(
+        'concept_lookup_dest_table',
+        action='store',
+        help='Table where suppressed concepts should be loaded')
+    retract_parser = subparsers.add_parser('retract')
+    retract_parser.add_argument(
+        '-s',
+        '--sandbox_dataset_id',
+        action='store',
+        dest='sandbox_dataset_id',
+        help='Identifies the dataset where output is stored',
+        required=True)
+
+    retract_parser.add_argument(
+        '-c',
+        '--concept_lookup_table',
+        action='store',
+        dest='concept_lookup_table',
+        help='Table where suppressed concepts are loaded',
+        required=True)
+
+    retract_parser.add_argument('-d',
+                                '--dataset_ids',
+                                action='store',
+                                dest='dataset_ids',
+                                nargs='+',
+                                help='Identifies dataset(s) to retract from',
+                                required=True)
+    return parser
+
+
+def setup(concept_lookup_table):
+    """
+    Load BQ tables needed before running
+
+    :param concept_lookup_table: destination table to save lookup
+    """
+    with open(DC732_CONCEPT_LOOKUP_CSV_PATH, 'rb') as csv_file:
+        job_config = bigquery.LoadJobConfig()
+        job_config.source_format = bigquery.SourceFormat.CSV
+        job_config.skip_leading_rows = 1
+        job_config.autodetect = True
+        job = CLIENT.load_table_from_file(csv_file,
+                                          destination=concept_lookup_table,
+                                          job_config=job_config)
+        job.result()
+        print("Loaded {} rows into {}.".format(job.output_rows,
+                                               concept_lookup_table))
+
+
+INVALID_TABLE_REF = (
+    "Invalid table reference: '{table}' "
+    "(qualify with either '<dataset>.<table>' or '<project>.<dataset>.<table>')"
+)
+
+
+def validate_lookup_table_arg(table):
+    """
+    Raise an error if the table reference is not qualified by dataset or project and dataset
+
+    :param table: reference to a table
+    """
+    dot_count = table.count('.')
+    if dot_count < 1 or dot_count > 2:
```

Comment:
opt:  so `if dot_count != 2`  ?

Reply:
It currently allows a) `dataset_id.table` or b) `project_id.dataset_id.table`. Do you think we should require b?